client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -208,111 +203,108 @@ export const PatternLibrary = ( {
 							<ToggleGroupControl
-								className="pattern-library__toggle--pattern-type"
-								isBlock
+								className="pattern-library__toggle--view"
 								label=""
-								onChange={ ( value ) => {
-									const href =
-										getCategoryUrlPath( category, value as PatternTypeFilter ) +
-										( isGridView ? '?grid=1' : '' );
-									recordClickEvent(
-										'calypso_pattern_library_type_switch',
-										currentView,
-										value as PatternTypeFilter
-									);
-									page( href );
-								} }
-								value={ patternTypeFilter }
+								isBlock
+								value={ isGridView ? 'grid' : 'list' }
 							>
 								<ToggleGroupControlOption
-									className="pattern-library__toggle-option"
-									label={ translate_not_yet( 'Patterns' ) }
-									value={ PatternTypeFilter.REGULAR }
+									className="pattern-library__toggle-option--list-view"
+									label={ ( <Icon icon={ iconMenu } size={ 20 } /> ) as unknown as string }
+									value="list"
+									onClick={ () => handleViewChange( 'list' ) }
 								/>
 								<ToggleGroupControlOption
-									className="pattern-library__toggle-option"
-									label={ translate_not_yet( 'Page layouts' ) }
-									value={ PatternTypeFilter.PAGES }
+									className="pattern-library__toggle-option--grid-view"
+									label={ ( <Icon icon={ iconCategory } size={ 20 } /> ) as unknown as string }
+									value="grid"
+									onClick={ () => handleViewChange( 'grid' ) }
 								/>
 							</ToggleGroupControl>
-						) }
-
-						<ToggleGroupControl
-							className="pattern-library__toggle--view"
-							label=""
-							isBlock
-							value={ isGridView ? 'grid' : 'list' }
-						>
-							<ToggleGroupControlOption
-								className="pattern-library__toggle-option--list-view"
-								label={ ( <Icon icon={ iconMenu } size={ 20 } /> ) as unknown as string }
-								value="list"
-								onClick={ () => handleViewChange( 'list' ) }
-							/>
-							<ToggleGroupControlOption
-								className="pattern-library__toggle-option--grid-view"
-								label={ ( <Icon icon={ iconCategory } size={ 20 } /> ) as unknown as string }
-								value="grid"
-								onClick={ () => handleViewChange( 'grid' ) }
-							/>
-						</ToggleGroupControl>
-					</div>
-
-					<PatternGallery
-						getPatternPermalink={ ( pattern ) =>
-							getPatternPermalink( pattern, category, patternTypeFilter, categories )
-						}
-						isGridView={ isGridView }
-						patterns={ patterns }
-						patternTypeFilter={ patternTypeFilter }
-					/>
-				</PatternLibraryBody>
-			) }
+						</div>
+
+						<PatternGallery
+							getPatternPermalink={ ( pattern ) =>
+								getPatternPermalink( pattern, category, patternTypeFilter, categories )
+							}
+							isGridView={ isGridView }
+							patterns={ patterns }
+							patternTypeFilter={ patternTypeFilter }
+						/>
+					</PatternLibraryBody>
+				) }
+			</div>
```

Comment:
Let's move this closing tag below the second `CategoryGallery`! Right now, the sticky header doesn't stay sticky when I scroll down to the page patterns

Reply:
Although I do see how this placement is a little bit nicer on the category pages 🤔 I still think we should move it, though

Reply:
Done

Reply:
>Although I do see how this placement is a little bit nicer on the category pages 🤔 I still think we should move it, though

Honestly, I suppose that this issue could be easy fixed after creating separate bodies for every page